src/Symfony/Bundle/SecurityBundle/DataCollector/SecurityDataCollector.php

```diff
@@ -123,6 +128,17 @@ public function collect(Request $request, Response $response, \Exception $except
             $this->data['voter_strategy'] = 'unknown';
             $this->data['voters'] = array();
         }
+
+        // collect firewall context informations
+        if ($this->firewallMap instanceof ContextAwareFirewallMapInterface) {
+            $firewallContext = $this->firewallMap->getContext($request);
+
+            if (null !== $firewallContext) {
+                $this->data['firewall'] = $firewallContext->getConfig();
+            }
+        } else {
+            $this->data['firewall'] = array();
```

Comment:
While my preference would be to set it to null, the twig check should already work with an empty array:
```php
> var_dump((bool) []);
bool(false)
```